core/Pos/Binary/Class/Instances.hs

```diff
@@ -1,299 +0,0 @@
-{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
This module was made redundant as all the instances for primitive types are now in `Pos.Binary.Class.Core`.